core/src/main/resources/lib/layout/graph.jelly

```diff
@@ -0,0 +1,56 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```

Comment:
copyright year and authors should be changed if this is for a new file.
`Jenkins contributors` if you don't want to put your own name